modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/view_cp_definition_grouped_entries.jsp

```diff
@@ -190,7 +190,11 @@ renderResponse.setTitle(cpDefinition.getName(themeDisplay.getLanguageId()));
 				'<liferay-ui:message key="are-you-sure-you-want-to-delete-the-selected-entries" />'
 			)
 		) {
-			var form = window.document['<portlet:namespace />fm'];
+			const form = window.document['<portlet:namespace />fm'];
```

Comment:
Okay I wasn't sure about these form specific selectors, thank you!

I remember something back in the day about there being only one form on the page and then stuff like this would fine, or something like that 🤷